common/meta/raft-store/src/state_machine/sm.rs

```diff
@@ -90,6 +90,42 @@ const SEQ_DATABASE_META_ID: &str = "database_meta_id";
 // const TREE_META: &str = "meta";
 const TREE_STATE_MACHINE: &str = "state_machine";
 
+pub enum DataChangeType {
+    Create,
+    Update,
+    Drop,
+}
```

Comment:
It may need a `Rename` operation type in it.

A rename is a bit different because it relates to two identifiers of an instance.